tests/test_main.py

```diff
@@ -337,3 +337,44 @@ class Config:
     with pytest.raises(ValueError) as exc_info:
         m.b = 11
     assert '"TestModel" object has no field "b"' in str(exc_info)
+
+
+def test_validating_assignment():
+    import pydantic
```

Comment:
you don't need this import

Reply:
The test fails if I remove it, as it uses `pydantic.constr`.

Reply:
So add `from pydantic import constr` to the top of the file.